pom.xml

```diff
@@ -399,6 +393,16 @@
       <artifactId>bootstrap-fileinput</artifactId>
       <version>${bootstrap-fileinput.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.webjars</groupId>
+      <artifactId>html5shiv</artifactId>
```

Comment:
This is from 2017.

Reply:
The problem is that the standard "jsf-api" of com.sun.faces is since 2019 no longer upgraded. We have two options. We can use another modern JSF implementation like "Jackarta JBoss" or just change to another technology. @thkoehler11 is of the opinion, that we should use another technology. In both cases, the effort is big. We need to discuss, what we do with the share client frontend.